assets/js/src/newsletter_editor/components/history.js

```diff
@@ -28,7 +29,14 @@ Module.HistoryView = Marionette.View.extend({
   },
 
   initialize: function initialize() {
+    var that = this;
     App.getChannel().on('afterEditorSave', this.addState, this);
+    Mousetrap.bind(['ctrl+z', 'command+z'], function keyboardUndo() {
+      that.undo();
+    });
```

Comment:
Can this be written just like `Mousetrap.bind(['ctrl+z', 'command+z'], this.undo)`?

Reply:
It can't :( Inside `undo`/`redo` methods, they are using `this`, which would be undefined if written as you wrote.